USER_MANUAL.md

```diff
@@ -385,5 +385,25 @@ Manual steps:<br>
   1. Start the gym again and note that the level actors should be at their previous shutdown positions, not their original positions. You can do this repeatedly.
   1. The test has now passed. Don’t forget to revert the two settings changes you made before you run another test.
 
+##### Ability Giving Gym
+Tests that ability specs given on different servers can be activated correctly via their handle.
```

Comment:
```suggestion
Tests that ability specifications given to an AbilitySystemComponent on two different servers can be activated correctly via their handles.
```

Reply:
Calling them specs here because the type name in the code is `FGameplayAbilitySpec`. Do you feel strongly about the written out word?

Reply:
No, in that case spec is fine. I'd still prefer that you include the rest of the changes in this snippet.

Reply:
Yea ofc